lisa/base_tools/service.py

```diff
@@ -200,3 +209,102 @@ def _check_service_running(self, name: str) -> bool:
 
 def _check_error_codes(cmd_result: ExecutableResult, error_code: int = 0) -> None:
     cmd_result.assert_exit_code(expected_exit_code=[0, error_code])
+
+
+class WindowsServiceStatus(int, Enum):
+    CONTINUE_PENDING = 5
+    PAUSE_PENDING = 6
+    PAUSED = 7
+    RUNNING = 4
+    START_PENDING = 2
+    STOP_PENDING = 3
+    STOPPED = 1
+    NOT_FOUND = 0
+
+
+class WindowsService(Tool):
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    @property
+    def command(self) -> str:
+        return ""
+
+    def enable_service(self, name: str) -> None:
+        pass
+
+    def restart_service(self, name: str, ignore_exit_code: int = 0) -> None:
+        try:
+            self.node.tools[PowerShell].run_cmdlet(
+                f"Restart-service {name}",
+                force_run=True,
+            )
+        except LisaException as identifier:
```

Comment:
The try/except can be removed totally. Just let the code raise the exception, instead of wrap it. Sam to stop_service.

Reply:
Removed.